src/services/textChanges.ts

```diff
@@ -83,14 +83,18 @@ namespace ts.textChanges {
         delta?: number;
     }
 
+    export interface ChangeNodesOptions {
+        nodesSeparator?: string;
+    }
+
     export type ChangeNodeOptions = ConfigurableStartEnd & InsertNodeOptions;
 
     interface Change {
         readonly sourceFile: SourceFile;
         readonly range: TextRange;
         readonly useIndentationFromFile?: boolean;
-        readonly node?: Node;
-        readonly options?: ChangeNodeOptions;
+        readonly node?: Node | Node[];

```

Comment:
I'm not sure I understand the type.  It's either `undefined`, one `Node`, zero `Node`s, a singleton list of `Node`s, or multiple `Node`s?